testdata/project-v4-declarative-v1/Dockerfile

```diff
@@ -32,9 +32,6 @@ RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o ma
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY --from=builder /workspace/manager .
-# copy channels
-COPY --from=builder /channels /channels
```

Comment:
Is this change related? Where would channels/ come under new layout? If its just `pkg/channels`, then shouldn't we copy that too or is there any other change in declarative config's layout?